main.py

```diff
@@ -109,6 +109,7 @@ def check_switch_consistency(self, switch):
             elif command == 'delete':
                 log.info('A consistency problem was detected in '
                          f'switch {dpid}.')
+                command = 'strict_delete'
```

Comment:
If the low-level command name is "delete_strict", why change it to "strict_delete"?

Reply:
Resolved at commit https://github.com/kytos/flow_manager/pull/119/commits/b0e7ee38786418261b5fd52502ff237a2c9467bc